extensions/amp-timeago/1.0/storybook/Basic.amp.js

```diff
@@ -33,7 +33,7 @@ storiesOf('amp-timeago', module)
         layout="responsive"
         width="100"
         height="100"
-        datetime={datetime}
+        datetime={String(datetime)}
```

Comment:
No idea why, but with `datetime={datetime}`, the DOM output is `datetime="{datetime}"`. Really weird behavior in storybooks.